officeInsightIndex.js

```diff
@@ -1,12 +1,11 @@
 var http = require('http');
 
-var PRODUCTION_HOST_AVAILABILITY = 'http://ec2-54-175-205-156.compute-1.amazonaws.com:8080/rest/';
-var MEETING_ROOM_AVAILABILITY = 'meetingRoom/availability/FIFTY_ONE';
+var PRODUCTION_HOST_AVAILABILITY = 'ec2-54-164-226-122.compute-1.amazonaws.com';
+var PRODUCTION_HOST_PORT = '8080';
+var MEETING_ROOM_AVAILABILITY = '/rest/meetingRoom/availability/FIFTY_ONE';
```

Comment:
were we going to do something about the different floors? I know that we are still just passing a static value of 'FIFTY_ONE' but are we going to also check the 53rd floor for rooms as well?